src/sparql/queries.py

```diff
@@ -4,11 +4,14 @@ def build_dataset_publisher_query() -> str:
         PREFIX foaf: <http://xmlns.com/foaf/0.1/>
         PREFIX dcat: <http://www.w3.org/ns/dcat#>
 
-        SELECT ?organizationNumber (COUNT(DISTINCT ?item) AS ?count)
+        SELECT ?organizationNumber (COUNT(DISTINCT ?dataset) AS ?count)
         FROM <https://datasets.fellesdatakatalog.digdir.no>
         WHERE {{
-            ?item a dcat:Dataset .
-            ?item dct:publisher ?publisher .
+            ?dataset a dcat:Dataset .
+            ?catalog dcat:dataset ?dataset .
+            OPTIONAL {{ ?dataset dct:publisher ?dsPublisher . }}
+            OPTIONAL {{ ?catalog dct:publisher ?catPublisher . }}
+            BIND (IF( EXISTS {{ ?dataset dct:publisher ?dsPublisher . }}, ?dsPublisher, ?catPublisher ) AS ?publisher ) .
```

Comment:
Muligens kan det være litt enklere med COALSCE, men ellers ser bra ut!
https://semaku.com/post/using-coalesce-and-if-in-sparql-for-nested/

Reply:
Prøvde med `COALESCE`. Fungerte bra, men med bare to utfall så var det lite forskjell fra en enkel IF, ble bare mer kode.
Kan slenge på et strategisk linjeskift, burde se litt bedre ut da.